rllib/evaluation/rollout_worker.py

```diff
@@ -74,7 +74,7 @@ def get_global_worker() -> "RolloutWorker":
     return _global_worker
 
 
-def _update_global_seed(policy_config: TrainerConfigDict, seed: int):
+def _maybe_update_global_seed(policy_config: TrainerConfigDict, seed: int):
```

Comment:
good idea.
I moved the global seeding function into debug.py. It became slightly more complicated, since I am passing all the global modules in.
also rebased on latest master.
let's see if tests are happy.
thanks.